org.mixedrealitytoolkit.data/CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [1.0.3-development] - 2024-04-17
+## [1.0.4-development] - 2024-04-18
+
+### Fixed
+
+* Fixed missing [CanEditMultipleObject] attributes as per Bug 573 [PR #698](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/698)
+
+## [1.0.3] - 2024-04-17
 
 ### Fixed
```

Comment:
Please double check all your version numbers.  For example org.mixedrealitytoolkit.data v1.0.3 is not released yet, so 1.0.4 is not needed.

Please referred to the last release branch here:

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/tree/releases/2024-03-20

Reply:
Done, I think, Can you please re-review? resolving this in the meantime.